pyvex/const.py

```diff
@@ -75,14 +80,19 @@ class U8(IRConst):
     c_constructor = pvc.IRConst_U8
 
     def __init__(self, value):
-        self.value = value
+        self._value = value
 
     def __str__(self):
         return "0x%02x" % self.value
 
     @staticmethod
     def _from_c(c_const):
-        return U8(c_const.Ico.U8)
+        global _U8_POOL
```

Comment:
You don't need to mark this name as global if you don't modify the object it references